x-pack/plugins/lists/public/exceptions/types.ts

```diff
@@ -59,7 +59,7 @@ export interface UseExceptionListProps {
 
 export interface ExceptionIdentifiers {
   id: string;
-  namespaceType: NamespaceType;
+  namespace_type: NamespaceType;
```

Comment:
Was this changed to snake case to match the request body? I would expect it to be `namespaceType` everywhere except the API call, which sends `namespace_type: namespaceType`

Reply:
Yea it was. Would I specify that change from `namespace_type` to `namespaceType` in the graphql types?

Reply:
Working to update this and a found bug.